connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/DefaultReplicationPolicy.java

```diff
@@ -43,6 +47,11 @@ public void configure(Map<String, ?> props) {
             log.info("Using custom remote topic separator: '{}'", separator);
             separatorPattern = Pattern.compile(Pattern.quote(separator));
         }
+
+        if (props.containsKey(INTERNAL_TOPIC_SEPARATOR_ENABLED_CONFIG)) {
+            log.info("Disable the usage of topic separator for internal topics");
```

Comment:
Is this always accurate? What if the user explicitly sets `replication.policy.internal.topic.separator.enabled` to `true`?

Also, some small wording suggestions:
```suggestion
            log.info("Disabling custom topic separator for internal topics; will use '.' instead of '{}'", separator);
```

Reply:
I think the first part of this still needs to be addressed? Specifically, we should only emit the log statement if `isInternalTopicSeparatorEnabled` is `false`.

Reply:
I only log it if `isInternalTopicSeparatorEnabled` is `false` now.

Reply:
Otherwise, we can remove it entirely.